src/main/resources/finalCopy/final_call_logs_dir_local.test

```diff
@@ -1,6 +1,7 @@
 # Tests that a runtime option to save call logs is copying said logs appropriately.
 
 name: final_call_logs_dir_local
+backends: [Local]
```

Comment:
I had not realized the WDL's were the same regardless of backend.  I'll re-use them.